src/LanguageServer/Protocol/Handler/InlayHint/InlayHintRefreshQueue.cs

```diff
@@ -33,18 +33,19 @@ public override void Dispose()
 
         private void OnOptionChanged(object? sender, OptionChangedEventArgs e)
         {
-            if (e.Option.Equals(InlineHintsOptionsStorage.EnabledForParameters) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForIndexerParameters) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForLiteralParameters) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForOtherParameters) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForObjectCreationParameters) ||
-                e.Option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatDifferOnlyBySuffix) ||
-                e.Option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatMatchArgumentName) ||
-                e.Option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatMatchMethodIntent) ||
-                e.Option.Equals(InlineHintsOptionsStorage.EnabledForTypes) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForImplicitVariableTypes) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForLambdaParameterTypes) ||
-                e.Option.Equals(InlineHintsOptionsStorage.ForImplicitObjectCreation))
+            if (e.HasOption(static option =>
+                option.Equals(InlineHintsOptionsStorage.EnabledForParameters) ||
+                option.Equals(InlineHintsOptionsStorage.ForIndexerParameters) ||
+                option.Equals(InlineHintsOptionsStorage.ForLiteralParameters) ||
+                option.Equals(InlineHintsOptionsStorage.ForOtherParameters) ||
+                option.Equals(InlineHintsOptionsStorage.ForObjectCreationParameters) ||
+                option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatDifferOnlyBySuffix) ||
+                option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatMatchArgumentName) ||
+                option.Equals(InlineHintsOptionsStorage.SuppressForParametersThatMatchMethodIntent) ||
+                option.Equals(InlineHintsOptionsStorage.EnabledForTypes) ||
+                option.Equals(InlineHintsOptionsStorage.ForImplicitVariableTypes) ||
+                option.Equals(InlineHintsOptionsStorage.ForLambdaParameterTypes) ||
+                option.Equals(InlineHintsOptionsStorage.ForImplicitObjectCreation)))
```

Comment:
indent

Reply:
Indent one more tab?

Reply:
yes please.  it helps see that it's in a lambda, as opposed to a check that is a sibling of the top check.  (i would like the indentation engine to do this automatically, but we've never gotten around to it).